x-pack/platform/plugins/shared/task_manager/server/task.ts

```diff
@@ -255,6 +256,40 @@ export interface IntervalSchedule {
    * An interval in minutes (e.g. '5m'). If specified, this is a recurring task.
    * */
   interval: Interval;
+  rrule?: never;
+}
+
+export interface RruleSchedule {
```

Comment:
This isn't "full" RRULE support, it appears, and that's fine.  But we'll need to document - presumably in the README for the plugin - what sorts of values we DO support.